jobs/views.py

```diff
@@ -47,8 +47,9 @@ class JobMixin:
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
 
-        active_locations = Job.objects.visible().distinct(
-            'location_slug'
+        active_locations = Job.objects.visible(
+        ).values_list('location_slug', flat=True
```

Comment:
Unfortunately, this will return

```py
<JobQuerySet ['lawrence-ks-us']>
```

instead of

```py
<JobQuerySet [<Job: Job Listing #6>]>
```

The output of `active_locations` will be used in `templates/jobs.base.html` so we also need to get `city`, `region`, and `country` fields.

Perhaps we change this line to

```py
values_list('location_slug', 'city', 'region', 'country')
```

?

Reply:
You're right, I misunderstood that.

What you propose makes sense, but as the data is not normalized, we'll have duplicate `location_slug` values if they have different `city`, `country`...

I think the only other option is to fetch all data from the database, and remove duplicate `location_slug` in Python.

Closing this PR.